docs/admin_menu/admin_menu.rst

```diff
@@ -556,9 +556,93 @@ Storages
+
+.. image:: quota_setting_page.png
+
+* Resource
+   * Usage: Shows the actual amount usage of the selected storage. 
+   * Endpoint: Represents the mount point of the selected storage.
+   * Backend Type: The type of storage.
+   * Capabilities: The supported feature of the selected storage.
+
+* Quota Settings
+   * For User: Configure per-user quota setting here.
+   * For Project: Configure per-project quota(project-folder) setting here.
+   * ID: Corresponds to user or project id.
+   * Hard Limit (GB): Currently set hard limit quota for selected quota.
+   * Control: Provides editing the hard limit or even deleting the quota setting.
+
+
+Set User Quota
+~~~~~~~~~~~~~~~
+
+In Backend.AI, there are two types of vfolders created by user and admin(project). In this section,
+we would like to show how to check current quota setting per-user and how to configure it.   
+First, Make sure the active tab of quota settings panel is ``For User``. Then, Select user you desire to 
+check and edit the quota. You can see the quota id that corresponds to user's id and the configuration already set 
+in the table, if you already set the quota.   
+
+.. image:: per_user_quota.png
+
+Of course, if you want to edit the quota, you can simply click the Edit button in the control column. After Clicking ``Edit`` button, you may see the small modal that enables configuring quota setting. 
+After input the exact amount, don't forget to Click ``OK`` button, unless the changes will not be applied.
+
+.. image:: quota_settings_panel.png
+
+
+Set Project Quota
+~~~~~~~~~~~~~~~~~~
+
+Setting a quota on project-folder is similar to setting a user quota. The difference between setting 
+project quota and user quota is to confirm setting the project quota requires one more procedure, 
+which is selecting the domain that the project is dependent on. The rest are the same.
+As in the picture below, you need to first select the domain, and then select the project.
+
+.. image:: per_project_quota.png
+
+
+.. image:: per_project_quota_2.png
+
+
+Unset Quota
+~~~~~~~~~~~~
+
+We also provides the feature to unset the quota. Please remind that after removing the quota setting, quota will automatically follows 
+user or project default quota, which cannot be set in WebUI. If you want to change the default quota setting, you may need to access to admin-only page.   
+By Clicking ``Unset`` button in control column, the small snackbar message will show up and confirm whether you really want to delete the current quota setting. 
+If you click ``OK`` button in the snackbar message, then it will delete the quota setting and automatically reset the quota follows to corresponding quota, 
+which depends on the quota type(user / project).
+
+.. image:: unset_quota.png
+
+
+.. note::
+   
+   When user/project doesn't have default value mapped to each uuid, 
+   then ``max_vfolder_size`` value will be the default value of each user/project.
```

Comment:
```suggestion
   If no user/project hard limit value is set for each UUID, the ``max_vfolder_size`` value in the user/project resource policy is used as the default value.
```

Reply:
Hmm, Wouldn't it be more natural to convert like this?
"If there's no config per user/project, then corresponding values in the user/project resource policy will be set as a default value. For example, If no hard limit value for quota is set, ``max_vfolder_size`` value in the resource policy is used as the default value."

What do you think @Sujin-Kim1? I considered the future implementation (e.g. soft limit, # of files, # of folders(directory)

Reply:
Super cool!